pgactivity/cli.py

```diff
@@ -1,15 +1,48 @@
+import logging
 import os
 import socket
 import sys
 import time
 from argparse import ArgumentParser
+from io import StringIO
+from typing import Optional
 
 from blessed import Terminal
 from psycopg2.errors import OperationalError
 
 from . import __version__, data, types, ui
 
 
+def configure_logger(debug_file: Optional[str] = None) -> StringIO:
+    logger = logging.getLogger("pgactivity")
+    logger.setLevel(logging.DEBUG)
+
+    # The steamhandler is used to print hints to the user at exit.
+    # The INFO log level is reserved for this.
```

Comment:
"Reserving" a log level for some purpose is rather fragile (in general, less here since we control everything). A better design would be define a custom logger for this:
```python
hint_logger = logging.getLogger("pgactivity-hints")
```